open-rosa/src/test/java/org/odk/collect/openrosa/http/OpenRosaPostRequestTest.java

```diff
@@ -138,7 +138,7 @@ public void sendsSubmissionFileAsFirstPartOfBody() throws Exception {
         assertThat(firstPartLines[1], containsString("name=\"xml_submission_file\""));
         assertThat(firstPartLines[1], containsString("filename=\"" + tempFile.getName() + "\""));
         assertThat(firstPartLines[2], containsString("Content-Type: text/xml"));
-        assertThat(firstPartLines[5], equalTo("<node>content</node>"));
+        assertThat(firstPartLines[4], equalTo("<node>content</node>"));
```

Comment:
I'm not quite sure that I understand what's happened here. Just different formatting from the version upgrade?